pkg/kubelet/token/token_manager.go

```diff
@@ -38,13 +40,14 @@ const (
 )
 
 // NewManager returns a new token manager.
-func NewManager(c clientset.Interface) *Manager {
+func NewManager(clients []clientset.Interface) *Manager {
 	m := &Manager{
-		getToken: func(name, namespace string, tr *authenticationv1.TokenRequest) (*authenticationv1.TokenRequest, error) {
-			if c == nil {
+		getToken: func(tenant, name, namespace string, ownerPodUID types.UID, tr *authenticationv1.TokenRequest) (*authenticationv1.TokenRequest, error) {
```

Comment:
Why add tenant as well as podUID?

Reply:
token param is needed for multi-tenant call. The func body used to call the single-tenanted call, which was defective.